bin/fin

```diff
@@ -5146,6 +5148,10 @@ config_get ()
 		target_env_file="$CONFIG_ENV"
 	else
 		load_configuration
+		if [[ -n $env ]]; then
+			shift
```

Comment:
First I wanted to write that shift is not needed, but then I understood why it is here. 
It was my mistake. 
We should use ${ARGV[0]} instead of $1 in line 5158, because parse_arguments have already parsed it into that array.
And then this shift will not be needed here